model_templates/training/python3_sklearn/push.yaml

```diff
@@ -0,0 +1,8 @@
+model:
+    name: drumpush-regression
+    type: training
+    targetType: regression
+    environmentID: 5e8c889607389fe0f466c72d
+client:
+    endpoint: https://staging.datarobot.com/api/v2/
+    token: NWVhNzNlMDQ3OTM1OWUxODQ3ZWMxMWFiOk5EdGxPN1lEYWU0NUtFc3lZMjhmMzhHTFdPWGI5L2ovT1dRajA5Yk5XWGs9
```

Comment:
Hi, 
Wait a sec.. lets have a discussion about metdata in DRUM models before we add anything.. 
This addition requires a PBMP or at least a design doc.. 
Please lets not do this  in  a PR without  a design discussion.
@yakov-g

Reply:
Although we discussed some aspects with @4dahalibut , but looks like @liororama is right and it is just too big for a PR.

Reply:
I wanted this PR to be the design discussion

Reply:
I could schedule a meeting if you'd like